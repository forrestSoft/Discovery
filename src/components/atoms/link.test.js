import React from 'react'
import {shallow, mount} from 'enzyme'
import shallowToJson from 'enzyme-to-json'

import Link from './link'

it('render matches snapshot', () => {
  const link = shallowToJson(shallow(
  	<Link 
      text="I am test helper text"
      url="google.com"
    />
  )) 
  expect(link).toMatchSnapshot();
});
