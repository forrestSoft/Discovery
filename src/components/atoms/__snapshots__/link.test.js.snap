// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render matches snapshot 1`] = `
<a
  href="google.com"
>
  I am test helper text
</a>
`;
